Orange/widgets/data/owtranspose.py

```diff
@@ -34,27 +39,41 @@ def __init__(self):
         super().__init__()
         self.data = None
 
-        # GUI
-        box = gui.vBox(self.controlArea, "Feature names")
-        self.feature_radio = gui.radioButtonsInBox(
-            box, self, "feature_type", callback=lambda: self.apply(),
-            btnLabels=["Generic", "From meta attribute:"])
+        box = gui.radioButtons(
+            self.controlArea, self, "feature_type", box="Feature names",
+            callback=lambda: self.apply())
+
+        button = gui.appendRadioButton(box, "Generic")
+        edit = gui.lineEdit(
+            gui.indentedBox(box, gui.checkButtonOffsetHint(button)), self,
+            "feature_name",
+            placeholderText="Prefix for column names",
```

Comment:
Currently, I can not read the whole placeholder text because the widget is too small and not resizable.
We can leave the text as is and just make the widget a bit larger?